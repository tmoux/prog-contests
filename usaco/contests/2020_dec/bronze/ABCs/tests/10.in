326552445 762589144 699462434 63126710 436036699 372909989 389679155
