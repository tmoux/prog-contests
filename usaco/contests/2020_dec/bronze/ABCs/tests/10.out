63126710 326552445 372909989
